app/vminsert/common/streamaggr.go

```diff
@@ -266,6 +266,9 @@ func pushAggregateSeries(tss []prompbmarshal.TimeSeries) {
 			}
 			ctx.AddLabel(name, label.Value)
 		}
+		if ctx.AreLabelsInvalid() {
```

Comment:
It's impossible case. 

 More over, it actually proves that all this changes for renaming `prompb` to `prompbmarshal` import is not needed.

 Please rollback it.

Reply:
done